examples/plot_enet_use_case.py

```diff
@@ -0,0 +1,157 @@
+"""
+===============================================================
+ElasticNet vs. Lasso: Illustration of ElasticNet use case
+===============================================================
+
+The Lasso estimator is useful in features selection thanks
+to its ability to produce sparse solutions. However,
+the latter becomes ineffective when it comes to high dimensional
+datasets as the support of the produced solution is limited
+by the number of samples.
+
+The ElasticNet estimator extend the feature selection property
+of Lasso estimator by producing sparse solutions regardless
+of the number of samples.
+
+Below is an example that illustrates that.
+"""
+import warnings
+
+from celer import ElasticNetCV, LassoCV
+from sklearn.model_selection import KFold
+
+import numpy as np
+from numpy.linalg import norm
+from scipy.linalg import block_diag
+import matplotlib.pyplot as plt
+
+
+# disable deprecation warning
```

Comment:
which deprecation warning do you get, we can try to fix its cause instead

Reply:
warning message
```bash
c:/Users/HP/Desktop/celer-repo/celer/examples/plot_enet_use_case.py:143: MatplotlibDeprecationWarning: Passing the linefmt parameter positionally is deprecated since Matplotlib 3.5; the parameter will become keyword-only two minor releases later.
  axs[i].stem(np.where(w)[0], w[w != 0], 'r', label=model_name,
```

solved by passing ``linefmt`` as keyword rather then positionally.

Reply:
:+1: